app/models/user.rb

```diff
@@ -19,6 +19,21 @@ class User < ApplicationRecord
 
   after_create :create_cart_list
 
+  def follow!(other_user)
+    raise ArgumentError if other_user == self
+
+    following << other_user
+  end
+
+  def unfollow!(following_user)
+    follow_relationship = active_relationships.find_by(followed_id: following_user.id)
+    follow_relationship&.destroy!
```

Comment:
follow_relationshipが存在しない場合の考慮がありますが、ない場合は例外を出してあげると良いと思います！
```suggestion
    follow_relationship = active_relationships.find_by(followed_id: following_user.id)
    raise ArgumentError if follow_relationship.nil?
    follow_relationship.destroy!
```